src/Shared/LayoutRenderers/AspNetResponseStatusCodeRenderer.cs

```diff
@@ -9,12 +11,25 @@ namespace NLog.Web.LayoutRenderers
     /// ASP.NET Response Status Code.
     /// </summary>
     /// <remarks>
-    /// <code>${aspnet-response-statuscode}</code>
+    /// <code>${aspnet-response-statuscode} emits the http status code as an int</code>
+    /// <code>${aspnet-response-statuscode:Format=value} where value is a valid enumeration format string emits the specified format</code>
```

Comment:
Can the docs about the possible format-values be moved down as `<remark>` for the `Format`-option?

Reply:
good suggestion, completed

Reply:
Thank you. It looks good.